packages/modules/gcloud/src/cloudstorage-emulator-container.ts

```diff
@@ -100,11 +111,7 @@ export class StartedCloudStorageEmulatorContainer extends AbstractStartedContain
    * @return a <code>host:port</code> pair corresponding to the address on which the emulator is
    * reachable from the test host machine.
    */
-  public getExternalUrl(): string {
-    if (this._externalURL) {
-      return this._externalURL;
-    } else {
-      return this.getEmulatorEndpoint();
-    }
```

Comment:
Removed this fallback because `undefined` is a valid state (i.e., "not set"), and `_externalURL` will usually be up-to-date, thanks to `updateExternalUrl`.